x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/reindex_job/reindex_job.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { EuiPageHeader, EuiPageContentBody } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { SetAppSearchChrome as SetPageChrome } from '../../../../shared/kibana_chrome';
+import { BreadcrumbTrail } from '../../../../shared/kibana_chrome/generate_breadcrumbs';
+
+interface Props {
+  schemaBreadcrumb: BreadcrumbTrail;
+}
+
+export const ReindexJob: React.FC<Props> = ({ schemaBreadcrumb }) => {
+  const { reindexJobId } = useParams() as { reindexJobId: string };
+
+  return (
+    <>
+      <SetPageChrome
+        trail={[
+          ...schemaBreadcrumb,
+          i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.reindexJob.breadcrumb', {
+            defaultMessage: 'Reindex job: {reindexJobId}',
+            values: { reindexJobId },
+          }),
+        ]}
```

Comment:
Hmmm I'm actually now rethinking this and wondering if I should change this breadcrumb copy/setup..... here it is right now with the rest of the page for context:

<img width="1147" alt="" src="https://user-images.githubusercontent.com/549407/116493858-71820e00-a854-11eb-9381-e2f36d3841f3.png">

But maybe this is better / more indicative of the page content:

<img width="548" alt="" src="https://user-images.githubusercontent.com/549407/116493503-bfe2dd00-a853-11eb-8613-74abf519239a.png">

Or maybe even this as a compromise?

<img width="772" alt="" src="https://user-images.githubusercontent.com/549407/116493512-c40efa80-a853-11eb-8046-998776ee346e.png">

WDYT?

Reply:
I don't mind it as it is. If you wanted to change it, I would opt for the second option over the third.

Reply:
Sweet! I think I'm gonna go for the 2nd option ✨

Reply:
```suggestion
        trail={[
          ...schemaBreadcrumb,
          i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.reindexErrorsBreadcrumb', {
            defaultMessage: 'Reindex errors',
          }),
        ]}
```